packages/lib-classifier/dev/components/App/App.js

```diff
@@ -173,6 +173,14 @@ class App extends React.Component {
     const mergedThemes = _.merge({}, baseTheme, zooTheme, { dark: this.state.dark })
     const key = this.state.cachePanoptesData ? 'cachedClassifier' : 'classifier'
 
+    const availableLocales = project?.configuration.languages.reduce((acc, current) => {
+      let newLocale = current
+      if (current === 'zh-cn') newLocale = 'zh-CN'
+      if (current === 'zh-tw') newLocale = 'zh-TW'
+      acc.push(newLocale)
+      return acc
+    }, [])
```

Comment:
This is also mapping one array to another, so `.map()` would be a better fit than `.reduce()`.
```suggestion
    const availableLocales = project?.configuration.languages.map(locale => {
      const [language, dialect] = locale.split('-')
      const newLocale = dialect ? `${language}-${dialect.toUpperCase()}` : language
      return newLocale
    })
```